exercises/practice/strain/tests/Tests.elm

```diff
@@ -21,9 +21,19 @@ isFirstLetter letter word =
     String.left 1 word == letter
 
 
-lessThanTen : Int -> Bool
-lessThanTen num =
-    num < 10
+alwaysTrue : a -> Bool
+alwaysTrue _ =
+    True
+
+
+alwaysFalse : a -> Bool
+alwaysFalse _ =
+    False
+
+
+containsFive : List Int -> Bool
```

Comment:
This is a nitpick, but you could remove the 'l' parameter here and make this point free